src/multirust-cli/setup_mode.rs

```diff
@@ -21,13 +21,20 @@ pub fn main() -> Result<()> {
              .help("Enable verbose output"))
         .arg(Arg::with_name("no-prompt")
              .short("y")
-             .help("Disable confirmation prompt."));
+             .help("Disable confirmation prompt."))
+        .arg(Arg::with_name("default-toolchain")
+             .short("d")
```

Comment:
Can we leave off the short option for now? Most flags don't have short options now, and I'd like to think more generally about how they should be used before adding them in bits and pieces.